### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by rabayarabu**
- Include bootstrap.
- Add styles for navbar, booklist and book form.
- Edit readme file.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @rabayarabu,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* Kindly fix these links in your readme file header. They are broken.

  fkfkfs
  dmdke

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.css [hunk 1]
```diff
@@ -1,56 +1,261 @@
...
+}
+
+.book {
+  font-family: 'Roboto Slab', sans-serif;
+  margin-block: 15px;
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
+  gap: 5px;
+}
```
Comment (brainconnect93) [inline src/App.css]: - [ ] Great effort working on the design of the Bookstore project, I would kindly suggest you stick to the original design as much as possible, Currently I notice the `profile icon` and the `add book` button does not align vertically on a straight line with the books added. Kindly check the original design to see how they all align vertically together on a straight line.
- [ ] Also the text in the `Add book` button is on a straight line in the original but what we have in your design is broken on two lines. Kindly work on it.

Your Design 👇 

image

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @rabayarabu, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @rabayarabu , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Clean code and green linters
✅ ... Great design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**